### ISSUE TITLE
Improve `\sqrt` parsing

### ISSUE THREAD

**Issue by GraemeWatt** [HDembinski/unicodeitplus#6]
```pyconsole
>>> print(parse(r'$\sqrt{s}$'))
√𝑠̅
>>> print(parse(r'$\sqrt{abc}$'))
√𝑎̅𝑏̅𝑐̅
```
In my opinion, this doesn't look good for different-sized letters inside the `\sqrt` argument.  There is also possible confusion with `\bar{s}` or `\overline{s}` which might have a different meaning.  I'd prefer that `\sqrt{s}` was replaced by just `√s` (if only one character as the argument of `\sqrt`) and `\sqrt{abc}` was replaced by `√(abc)` (if more than one character as the argument of `\sqrt`), i.e. use round brackets to indicate the grouping in the latter case.

Here are a couple of more complicated examples that currently give strange results from `unicodeitplus`:
1. `Pb+Pb collisions at $\sqrt{s_{NN}}=2.76$ TeV with the ATLAS detector`
2. `$\sqrt{s_{_{NN}}}=200$ GeV`

Encodings like this are very common in the titles of heavy-ion physics publications and so we need sensible output from `unicodeitplus`.

**Comment by HDembinski** [comment]
Agreed.

**Comment by HDembinski** [comment]
`$\sqrt{s_{_{NN}}}=200$ GeV`

That's an interesting case. We cannot support double subscript in unicode. It is not clear how that should be handled with a general rule. In this particular case, it is clear that one can ingore the second subscript and get an acceptable result, but people may also write something like this `v_{K_0^S}`.

**Comment by GraemeWatt** [comment]
I agree that `$\sqrt{s_{_{NN}}}=200$ GeV` could be treated in the same way as `$\sqrt{s_{NN}}=200$ GeV` giving Unicode output `√𝑠ₙₙ=200 GeV`.  Currently (in v0.3.1) it gives `√𝑠̅_{ₙ}̅_{ₙ}̅=200 GeV`.

For cases where it is difficult to translate to Unicode like `$v_{K_0^S}$` I prefer that you just return the original LaTeX encoding.  This is preferable to producing mangled output like `𝑣ₖ_{₀}_{^{S}}` (in v0.3.1).  Alternatively, you could raise an exception, leaving it to be handled by the user.  I agree that incorrect LaTeX encodings (#8) should raise an exception.

**Comment by HDembinski** [comment]
Yes `√𝑠̅_{ₙ}̅_{ₙ}̅=200 GeV` is wrong and this is a bug in the parser. My goal is also to produce `√𝑠ₙₙ`, which requires a new special case for how `\sqrt{...}` is handled. This is work in progress.

> For cases where it is difficult to translate to Unicode like $v_{K_0^S}$ I prefer that you just return the original LaTeX encoding. 

That's not my preference. The goal of unicodeitplus is to find a reasonable approximation if an exact mapping is not possible. So I would return `𝑣ₖ₀ₛ` in this case rather than the raw LaTeX.

**Comment by HDembinski** [comment]
I assume that we will always find a counter example to any rule that we apply when an exact conversion is not possible. It is a trade-off. If the rule I have in mind and sketched above turns out to produce many of these counter examples, I will reconsider and rather return the raw LaTeX in this case.
